src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -76,35 +76,31 @@ private static void runConditionalMutationsTest(Environment env, String tableNam
     ConditionalWriter cw = env.getClient().createConditionalWriter(tableName,
         new ConditionalWriterConfig());
 
-    conditionalMutationsTime(cw, null, "0", reportBuilder);
+    conditionalMutationsTime(cw, null);
 
     double rateSum = 0.0;
     for (int i = 1; i < 20; i++) {
-      rateSum += conditionalMutationsTime(cw, (long) i, ((Integer) i).toString(), reportBuilder);
+      rateSum += conditionalMutationsTime(cw, (long) i);
```

Comment:
Could just change the type of `i` to `long`... since it's going to be cast anyway.